central/convert/storagetov2/scan.go

```diff
@@ -0,0 +1,123 @@
+package storagetov2
```

Comment:
These `convert` functions look like they should be part of the first Central PR.